irr/src/CNullDriver.h

```diff
@@ -309,7 +309,7 @@ class CNullDriver : public IVideoDriver, public IGPUProgrammingServices
 			}
 		}
 
-		virtual ~SHWBufferLink()
+		~SHWBufferLink()
```

Comment:
Wouldn't this change make it more error-prone to leak memory if a destructor is defined within a derived class?

Reply:
there's no derived destructor currently, but yes I'll undo this.